src/Controller/AdminController.php

```diff
@@ -52,4 +60,79 @@ public function partners(): Response
     {
         return $this->render('admin/partners.html.twig');
     }
+
+    /**
+     * Displays the page for add a new charging-station
+     * @Route("borne-de-recharge/ajouter", name="charging_station_new", methods={"GET","POST"})
+     * @return Response
+     */
+    public function newChargingStation(Request $request): Response
+    {
+        $refillStation = new RefillStation();
+        $form = $this->createForm(RefillStationType::class, $refillStation);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager = $this->getDoctrine()->getManager();
```

Comment:
According to symfony best practices, you should use dependency injection for the entity manager